proxies/rest/Dockerfile

```
@@ -41,7 +41,7 @@ COPY --from=builder /app/dist /app/dist
 COPY --from=prod-deps /app/node_modules /app/node_modules
 WORKDIR /app
 # copy necessary files
-COPY package.json ./
+COPY package.json READNE.md ./
```

Btw do we really need to copy that?

```suggestion
COPY package.json README.md ./
```

---

sorry didnt saw you fixed it